src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -28175,7 +28181,9 @@ static void Main(string[] args)
             var inFieldDeclaratorArgumentlist = declarator != null && declarator.Parent.Parent.Kind() != SyntaxKind.LocalDeclarationStatement &&
                                            (declarator.ArgumentList?.Contains(decl)).GetValueOrDefault();
 
-            AssertInfoForDeclarationExpressionSyntax(model, decl);
+            // We're not able to get type information at such location (out var argument in global code) at this point
+            // Seee https://github.com/dotnet/roslyn/issues/13569
+            AssertInfoForDeclarationExpressionSyntax(model, decl, expectType: false);
```

Comment:
>expectType: false [](start = 66, length = 17)

Semantic model should work outside of VariableDeclaratorSyntax.ArgumentList syntax. Please adjust verification accordingly, see ```inFieldDeclaratorArgumentlist```.